### PR TITLE
Update documentation based on various feedback (QC-313)

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

**Comment by Barthelemy** [comment]
HMPID and QC-313

File: doc/QuickStart.md [hunk 1]
```diff
@@ -19,6 +19,14 @@
 
 [↑ Go to the Table of Content ↑](../README.md) | [Continue to Modules Development →](ModulesDevelopment.md)
 
+## Read this first !
```
Comment (knopers8) [inline doc/QuickStart.md:22]: To be extremely picky, in English there is no need for a space before an exclamation mark.

File: doc/QuickStart.md [hunk 2]
```diff
@@ -130,12 +142,12 @@ enabled=1
 (...)
 ```
 
-Start Readout :
+Start Readout in a terminal :
```
Comment (knopers8) [inline doc/QuickStart.md:145]: Nor we need spaces before semicolons.

File: doc/QuickStart.md [hunk 3]
```diff
@@ -19,6 +19,14 @@
 
 [↑ Go to the Table of Content ↑](../README.md) | [Continue to Modules Development →](ModulesDevelopment.md)
 
+## Read this first !
+
+This page will give you a basic idea of the QC and how to run it. Please read it *in its entirety* and run the commands along the way. Do not start developing your module before you have reached the next section called "Modules Development".
```
Comment (knopers8) [inline doc/QuickStart.md:24]: Oh, one idea here. If you agree with me, can you indicate that one should pull the latest QC version if they read the latest documentation?

File: doc/QuickStart.md (hunk 1 above)
Comment (Barthelemy) [reply doc/QuickStart.md:22]: correct, thanks !   
correct, thanks!

File: doc/QuickStart.md (hunk 2 above)
Comment (Barthelemy) [reply doc/QuickStart.md:145]: done

File: doc/QuickStart.md (hunk 3 above)
Comment (Barthelemy) [reply doc/QuickStart.md:24]: good idea!

**Pull request merged**